.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
nock in node 18 was doing something really funky with websockets and the tests would just hang

node18 is out of LTS so I think its ok to put 20 as out min version